mava/utils/jax_training_utils.py

```diff
@@ -15,9 +15,89 @@ def action_mask_categorical_policies(
         distribution.logits,
         jnp.finfo(distribution.logits.dtype).min,
     )
+
     return tfd.Categorical(logits=masked_logits, dtype=distribution.dtype)
 
 
+def compute_running_mean_var_count(stats: Array, batch: Array) -> Array:
+    """Updates the running mean, variance and data counts during training.
+
+    stats (array) -- mean, var, count.
+    batch (array) -- current batch of data.
+
+    Returns:
+        stats (array)
+    """
+
+    batch_mean = jnp.mean(batch)
+    batch_var = jnp.var(batch)
+    batch_count = batch.size
+
+    mean, var, count = stats
+
+    delta = batch_mean - mean
+    tot_count = count + batch_count
+
+    new_mean = mean + delta * batch_count / tot_count
+    m_a = var * count
+    m_b = batch_var * batch_count
+    M2 = m_a + m_b + jnp.square(delta) * count * batch_count / tot_count
+    new_var = M2 / tot_count
+    new_count = tot_count
+
+    return jnp.array([new_mean, new_var, new_count])
+
+
+def dummy_running_mean_var_count(stats: Array, batch: Array) -> Array:
```

Comment:
Since this is getting used in the tests can we please define this function in one of the testing folders or even in the test where it is getting used?

Reply:
It doesn't do anything specific but I will add a test for it for completeness.

Reply:
@RuanJohn I have pushed the test